src/core/load-fragment.c

```diff
@@ -2860,7 +2860,15 @@ int config_parse_device_allow(
                 return 0;
         }
 
-        n = strcspn(rvalue, WHITESPACE);
+        int r;
+        if((r = unit_name_printf(userdata, rvalue, &t)) < 0) {
+                log_syntax(unit, LOG_ERR, filename, line, -r,
```

Comment:
While some of the code still does this, you can actually pass `r` instead of `-r` here, the call will negate as necessary. Please drop the negation in new code.

Please dongrade this from LOG_ERR to LOG_WARNING, as we proceed here, and do not fail the operation.